nvflare/app_opt/sklearn/data_loader.py

```diff
@@ -42,13 +42,10 @@ def get_pandas_reader(data_path: str):
 
 def load_data(data_path: str, require_header: bool = False):
     reader = get_pandas_reader(data_path)
-    if hasattr(reader, "header"):
-        if require_header:
-            data = reader(data_path)
-        else:
-            data = reader(data_path, header=None)
-    else:
+    if hasattr(reader, "header") and require_header:
         data = reader(data_path)
+    else:
+        data = reader(data_path, header=None)
```

Comment:
One of the "reader" that we get from "get_pandas_reader" is "read_json",
That method does not take in argument "header", I think we can't pass the "header" argument to it.

Reply:
@mikez3 thanks for the PR, can you remove the "read_json" in line 21 in this PR?

Reply:
Okay, it's done

Reply:
Thanks!